pygerrit/rest/__init__.py

```diff
@@ -42,6 +42,8 @@ def _decode_response(response):
 
     """
     content = response.content.strip()
+    if not isinstance(content, str):
```

Comment:
This was fixed slightly differently in `pygerrit2`.  See https://github.com/dpursehouse/pygerrit2/blob/master/pygerrit2/rest/__init__.py#L49